model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -327,6 +327,11 @@ message StandardPTransforms {
     //
     // Payload: none
     TO_STRING = 8 [(beam_urn) = "beam:transform:to_string:v1"];
+
+    // Specialized implementation of GroupByKey for translating Redistribute transform into
+    // Dataflow service protos.
+    DATAFLOW_GROUP_BY_KEY = 9
```

Comment:
I don't think you need this, because you should not have this transform in a portable graph, only in the v1beta3 (which is a pure Java override straight to v1beta3 and exists without the proto)